core/CronArchive/FixedSiteIds.php

```diff
@@ -1,65 +0,0 @@
-<?php
-/**
- * Piwik - free/libre analytics platform
- *
- * @link https://matomo.org
- * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
- *
- */
-namespace Piwik\CronArchive;
-
-
-class FixedSiteIds
```

Comment:
Are they no longer needed @diosmosis ?

Reply:
No longer needed. Parallelization is done by idarchive not by site (though we go in order of sites to mimic old behavior).